operator/v1/types_network.go

```diff
@@ -480,6 +486,14 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type IPsecExternal struct {
+	// State controls the node's external (aka NS) ipsec service state.
```

Comment:
Comments should start with the JSON tag name.

I'd appreciate if you could expand the documentation on this field, please review the API conventions on [writing good godocs](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#write-user-readable-documentation-in-godoc) and expand this with relevant information.